iam.tf

```diff
@@ -33,7 +33,15 @@ resource "aws_iam_role_policy" "ext_tableau" {
         "arn:aws:ssm:eu-west-2:*:parameter/ext_tableau_hostname",
         "arn:aws:ssm:eu-west-2:*:parameter/tableau_server_username",
         "arn:aws:ssm:eu-west-2:*:parameter/tableau_server_password",
-        "arn:aws:ssm:eu-west-2:*:parameter/tableau_ext_s3_prefix"
+        "arn:aws:ssm:eu-west-2:*:parameter/tableau_ext_s3_prefix",
+        "arn:aws:ssm:eu-west-2:*:parameter/data_archive_tab_ext_backup_url",
+        "arn:aws:ssm:eu-west-2:*:parameter/tab_ext_repo_url",
+        "arn:aws:ssm:eu-west-2:*:parameter/tab_ext_repo_host",
+        "arn:aws:ssm:eu-west-2:*:parameter/tab_ext_repo_port",
+        "arn:aws:ssm:eu-west-2:*:parameter/tableau_admin_username",
+        "arn:aws:ssm:eu-west-2:*:parameter/tableau_admin_password",
```

Comment:
Is internal/external tableau admin will use the same password?

Reply:
Yes, that was the plan.

Reply:
👍